### ISSUE TITLE
notification link of welcome message not working ...

### ISSUE THREAD

**Issue by dogi** [ole-vi/planet#455]
![screen shot 2018-02-28 at 1 33 00 am](https://user-images.githubusercontent.com/13684059/36773427-974a783c-1c27-11e8-95d1-82fa5ee1e586.png)

**Comment by paulbert** [comment]
Yes, the initial welcome notification does not have a link.  It's being added in #443, so clicking on the welcome notification will take you to the notification list that is the main part of the PR.
